lib/fizzy/parser_expr.cpp

```diff
@@ -148,6 +148,11 @@ parser_result<Code> parse_expr(
     // The function's implicit block.
     control_stack.emplace(Instr::block, function_arity, 0);
 
+    code.local_count = local_count;
+
+    // This is needed for aiding validation.
+    local_count += static_cast<uint32_t>(module.typesec[func_type_idx].inputs.size());
```

Comment:
Okay to increment the argument or should have a new temporary?

Reply:
It is better to have separate variable (I find modifying args values confusing). But I have not suggestions how to name the new variable.

Reply:
`max_local_index` ?

Reply:
Done.